jupyterhub/apihandlers/users.py

```diff
@@ -209,14 +218,16 @@ def post(self, name):
             raise web.HTTPError(400, "%s's server is not running" % name)
         self.set_server_cookie(user)
         # a service can also ask for a user cookie
-        # this code prevent to raise an error
+        # this code prevents to raise an error
+        # cause service doesn't have 'other_user_cookies'
         if getattr(current, 'other_user_cookies', None) is not None:
             current.other_user_cookies.add(name)
 
 
 default_handlers = [
     (r"/api/users", UserListAPIHandler),
     (r"/api/users/([^/]+)", UserAPIHandler),
-    (r"/api/users/([^/]+)/server", UserServerAPIHandler),
+    (r"/api/users/([^/]+)/servers", UserCreateServerAPIHandler),
```

Comment:
It would only be allowed when multiple servers are not allowed, so there cannot be ambiguity between `/user/name` and `/user/name/1/`. So the condition is either:

- require exactly one server and that it has no name
- require multiple server and that they have names